File: Java/Day1W.java
package Daywise;

import java.util.ArrayList;

public class Day1W {
    public static void main(String[] args){
        String paragraph = "Bold hit an bat";
        int j=0;
        ArrayList<String> str = new ArrayList<>();
        int start =0;
        while(j<paragraph.length()){
            char ch = paragraph.charAt(j);
            if(ch==32){
                str.add(paragraph.substring(start,j));
                start=j+1;
            }
            j++;

        }
        str.add(paragraph.substring(start));

        System.out.println(str);

    }
}

File: Python/Day1W.py
paragraph = "Bold Hit an hat"
j=0
words = []
temp=""
while(j<len(paragraph)):
    if(paragraph[j]==" "):
        words.append(temp)
        temp=""
        j+=1
        continue
    temp+=paragraph[j]
    #print(paragraph[j],end="")
    j+=1
words.append(temp)
print(words)